File: src/main/java/com/sm/carwashmonitor/dto/StationStatisticsDTO.java
package com.sm.carwashmonitor.dto;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class StationStatisticsDTO {
    String stationName;
    Float waterCost;
    Float detergentCost;
    Float waxCost;
    Float revenue;
}

File: Dtos/StationStatisticsDto.cs
namespace CarwashMonitor.Dtos
{
    public class StationStatisticsDto
    {
        public string? StationName { get; set; }
        public float WaterCost { get; set; }
        public float DetergentCost { get; set; }
        public float WaxCost { get; set; }
        public float Revenue { get; set; }
    }
}
